### DISCUSSION TITLE
Detect when in copy mode

### DISCUSSION THREAD

**Discussion by Taaqif** [wezterm/wezterm#2345] [Q&A] [answered]
I've been using wezterm for a while now and love the customisation available. 

I was wondering if there is an API method to detect when in copy mode?
My intent it to update the color of the tab when copy mode is activated to make it obvious that the mode is enabled

Thanks again for the great product

**Comment by wez** [comment, marked as answer]
probably the simplest thing to check is: the pane title for the pane in the format-tab-title event will start with `Copy mode:`
https://wezfurlong.org/wezterm/config/lua/PaneInformation.html
